run_ml.py

```diff
@@ -1,20 +1,34 @@
 #!/usr/bin/env python3
+import os
 import sys
-import util
+from util import util
 import requests
-from collections import defaultdict
 import pandas as pd
+import pickle
 
 CONSTANT__OUTSIDE = "OUTSIDE"  # enum from graphql-gateway; if it changes, the extraction service breaks!
 
 
-def run_classification(corpus_embeddings, corpus_labels, corpus_ids, training_ids):
-    from active_transfer_learning import ATLClassifier
+def run_classification(
+    information_source_id,
+    corpus_embeddings,
+    corpus_labels,
+    corpus_ids,
+    training_ids,
+):
+    from util.active_transfer_learning import ATLClassifier
```

Comment:
is there a specific reason this isn't in the import block at the beginning?

Reply:
I don't know. The ATLClassifier is appended to the active_transfer_learning module, when the ml-exec-env container is run. Therefore, the ATLClassifier only exists at the runtime of the container, but prior to the runtime of the python script. I think, it should also work, when it is imported at the beginning.

Reply:
we can leave it as is. didn't know it was something runtime related. Thought since it's static here we would have it static imported